server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/ScanDataSource.java

```diff
@@ -186,12 +188,28 @@ public boolean isCurrent() {
 
     if (!loadIters) {
       return visFilter;
-    } else if (null == options.getClassLoaderContext()) {
-      return iterEnv.getTopLevelIterator(IteratorUtil.loadIterators(IteratorScope.scan, visFilter, tablet.getExtent(), tablet.getTableConfiguration(),
-          options.getSsiList(), options.getSsio(), iterEnv));
     } else {
-      return iterEnv.getTopLevelIterator(IteratorUtil.loadIterators(IteratorScope.scan, visFilter, tablet.getExtent(), tablet.getTableConfiguration(),
-          options.getSsiList(), options.getSsio(), iterEnv, true, options.getClassLoaderContext()));
+      List<IterInfo> iterInfos;
+      Map<String,Map<String,String>> iterOpts;
```

Comment:
Can you describe what is going on in this block?

Reply:
I added some comments